engineering_tools/carma_script_extensions/swarm

```diff
@@ -60,6 +59,14 @@ swarm__install() {
 
 swarm__start() {
     echo "Checking Docker Swarm status..."
+    # if carma-config-data volume doesn't exist exit and request update
+    if [ ! -z "$(docker volume ls | grep carma-config-data)" ]; then
+        echo "Found 'carma-config-data' volume"
+    else
+        echo "Missing 'carma-config-data' volume, add with \"carma swarm update-config <image tag>\""
```

Comment:
Should this output to the stderr instead of stdout?

Reply:
Will first need to add an error function in the top level carma script. Not in scope but will track